packages/terra-clinical-result/src/common/utils.js

```diff
@@ -25,6 +25,34 @@ const checkTypeNumeric = (resultData) => {
   return isNumeric;
 };
 
+/*
+ * Returns a copy of the object with additional parameters with trimmed values and informational booleans
+ * @param {object} observationPropShape
+ * @return {object} is an object with added trimmed values and information.
+ */
+const createResult = (data) => {
+  if(!data) {
+    return {};
+  }
+  const returnResult = JSON.parse(JSON.stringify(data));
```

Comment:
Could we use data directly without JSON.parse & JSON.stringify?

Reply:
if we use `data` directly we start adding to the original data source of `systolic` and `diastolic`. I was trying to find a way to clone the object that worked in all browsers.

Reply:
did you try `{ ...data }`?

Reply:
Well now I feel dumb......
Yea that works.

Reply:
Fixed now https://github.com/cerner/terra-clinical/pull/648/commits/bcd9623974696cc69c750cf77514f41a424ac509#diff-72ca9008b342e1caa5acc1466dfa7f20R37